src/components/Avatar.tsx

```diff
@@ -16,13 +16,16 @@ limitations under the License.
 
 import React, { useEffect, useState } from 'react';
 import classNames from 'classnames';
-import { Group, Room, User } from '../matrix-cypher';
 
-import { getMediaQueryFromMCX } from '../utils/cypher-wrapper';
+import { Group, Room, User } from '../matrix-cypher';
+import useHSs from '../utils/getHS';
+import { getThumbnailURI } from '../utils/cypher-wrapper';
 import logo from '../imgs/chat-icon.svg';
 
 import './Avatar.scss';
 
+const AVATAR_SIZE = 60;
```

Comment:
I'd suggest requesting 96 to get a crisper image (and to match what clients already request and thus should have cached server-side)

Reply:
Thanks for the tip